quiche/balsa/balsa_headers.cc

```diff
@@ -1102,7 +1102,7 @@ void BalsaHeaders::RemoveLastTokenFromHeaderValue(absl::string_view key) {
       GetHeaderLinesIterator(key, header_lines_.begin());
   if (it == header_lines_.end()) {
     QUICHE_DLOG(WARNING)
-        << "Attempting to remove last token from a non-existent "
+        << "Attempting to remove last token from a nonexistent "
```

Comment:
Happy to drop this or anything else -- in general, I group changes by the resulting spelling so if you don't like a correction, you can flag it once and I'll drop the commit.